src/shared/components/oncoprint/DeltaUtils.spec.ts

```diff
@@ -36,15 +39,24 @@ describe('Oncoprint DeltaUtils', () => {
             assert.equal(
                 numTracksWhoseDataChanged(
                     [],
-                    [{ key: 'a', data: [] }, { key: 'b', data: [] }]
+                    [
+                        { key: 'a', data: [] },
+                        { key: 'b', data: [] },
+                    ]
                 ),
                 2,
                 'tracks deleted'
             );
         });
         it('should return 3 for one track deleted, one track added, one track changed', () => {
-            let state1 = [{ key: 'a', data: [] }, { key: 'b', data: [] }];
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki